android/java/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/views/MapView.java

```diff
@@ -3116,6 +3133,36 @@ public void setOnMyLocationChangeListener(@Nullable OnMyLocationChangeListener l
         mUserLocationView.setOnMyLocationChangeListener(listener);
     }
 
+    /**
+     * Set the current user location tracking mode.
+     *
+     * @param userLocationTrackingMode to be used.
+     */
```

Comment:
Can you perhaps add a bit more detail on what this does from app developers perspective, and add a `{@link }` and a `@see` to the mode constants.

Also for the `is`/`setMyLocationEnabled` the current phrasing is "my-location" instead of user. So either change this one to match or change them both to something you think fits better.